### ISSUE TITLE
"memory access out of bounds" error when decoding point cloud to quantized form

### ISSUE THREAD

**Issue by lilleyse** [google/draco#402]
I'm getting a "memory access out of bounds" error when decoding very small point clouds when skipping the attribute transform.

If `dequantizeInShader` is true, `DecodeBufferToPointCloud` will fail, but only for small files.

```javascript
function decodeDracoPointCloud(draco, data, dequantizeInShader) {
    var dracoDecoder = new draco.Decoder();

    if (dequantizeInShader) {
        dracoDecoder.SkipAttributeTransform(draco.POSITION);
    }

    var buffer = new draco.DecoderBuffer();
    buffer.Init(data, data.byteLength);

    var dracoPointCloud = new draco.PointCloud();
    dracoDecoder.DecodeBufferToPointCloud(buffer, dracoPointCloud);
}
```

I put together a small sample here: 
[draco-bug.zip](https://github.com/google/draco/files/2031965/draco-bug-3.zip)

In the folder run npm install and npm start. Go to localhost:8003 and look at the console.

This error comes up when using both the WebAssembly and asm.js decoder. I'm using 1.3.1.

**Comment by ondys** [comment]
I believe I found the issue, but just to make sure, was the draco-bad.bin encoded without quantization? From debugging, it seems like the position attribute was not quantized which is causing the problem when the skip attribute transform is requested. If that's the case, the fix I'm working on will solve the crashing, but there will be no attribute transform available for the position attribute (since it wasn't transformed).

**Comment by lilleyse** [comment]
I printed out the quantization bits and it was in fact 0 for these small tiles. Thanks for noticing that.

**Comment by ondys** [comment]
fixed in 1.3.3
